### ISSUE TITLE
Add configuration options as parameters to `scaffold`

### ISSUE THREAD

**Issue by svallory** [milesj/packemon#241]
Hi there! Thanks for the awesome tool :)

I'm automating the creation of packages inside my company and would like to pass `author`, `project-name`, and `repositoryUrl` to the `scaffold` command.

Also, I'd like to suggest that inside monorepos that are already git repositories the repository URL could be automatically detected ;)

**Comment by milesj** [comment]
@svallory We already prompt for these fields. Do you mean with an arg like `--author`?

**Comment by svallory** [comment]
> @svallory We already prompt for these fields. Do you mean with an arg like `--author`?

Exactly. I want to be able to use packemon in a fully automated way so I can standardize package creation in my organization, and so I don't have to fill them in all the time.
